### ISSUE TITLE
Hvap is None for apparently valid inputs.

### ISSUE THREAD

**Issue by cfcohen** [jjgomera/iapws#37]
This is probably just a documentation problem combined with my lack of knowledge about IAPWS, but I can't find anything in the documentation to explain why this:

iapws.IAPWS95(T=298.15, P=1.0).Hvap

returns None.

In fact the documentation implies that if IAPWS95 is initialized with two parameters (e.g. T & P) then Hvap will be defined.  The property calculable is set to True.

Line 875 of iapws95.py sets HVap to None unless 0<x<1, but setting x to any practically any value in that range just results in tracebacks.   For example:

$ python3 -c "import iapws; print(iapws.IAPWS95(T=274.15, P=1, x=0.5))"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 167, in __init__
    self.__call__(**kwargs)
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 186, in __call__
    self.calculo()
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 843, in calculo
    self.fill(self.Liquid, liquido)
UnboundLocalError: local variable 'liquido' referenced before assignment

$ python3 -c "import iapws; print(iapws.IAPWS95(T=274.15, P=1, x=1))"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 167, in __init__
    self.__call__(**kwargs)
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 186, in __call__
    self.calculo()
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 840, in calculo
    self.fill(self.Gas, propiedades)
  File "/usr/lib/python3/dist-packages/iapws/iapws95.py", line 897, in fill
    fase.rho = estado["rho"]
TypeError: 'NoneType' object is not subscriptable

Neither of these results seem to intended.

**Comment by jjgomera** [comment]
Hi,
Hvap is only defined in two phase point states. So simple as the different between the enthalpy in both phases. So really Hvap is only a function of saturation temperature (or the pressure de saturation):

```
In [1]: import iapws
In [2]: st = iapws.IAPWS95(T=298.15, x=0.5)
In [3]: st.Hvap
Out[3]: 2441.6761749130196
In [4]: st.Gas.h-st.Liquid.h
Out[4]: 2441.6761749130196
```
With T-P as input pair it can't possible define a point in the two phase region

The errors you find is a bug, solved in last commit, try it.

**Comment by cfcohen** [comment]
Thanks!  I vaguely understood that it was a phase problem, and that passing x should correct the problem, but I couldn't figure out how to make it work.  I am using IAPWS not because I'm a serious "water scientist", but because it seemed like an easy way to get an accurate value for various properties (e.g. enthalpy or the speed of sound) across a broad range of values without having to worry about validity ranges.  Most other other formulas I've found have very limited validity ranges.   In the future, I may report a few other issues caused by my improper use of the package or ignorance of the subject matter, since I've generated tracebacks on a couple of occasions by doing stupid things.   I feel better knowing that this was a fixable bug, which I've tested and can confirm that it's working now. ;-)
